x-pack/plugin/rollup/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/rollup/60_settings.yml

```diff
@@ -91,6 +91,275 @@
   - match: { test-rollup.settings.index.default_pipeline: null }
   - match: { test-rollup.settings.index.final_pipeline: null }
 
+---
+"Downsample index with tier preference":
```

Comment:
I wrote different tests because in the two scenarios the default value we are overwriting is different. In one case it is "data_content" while in the other is "data_hot". My understanding is that this setting is applied differently and I wanted to make sure we are ALWAYS overwriting that.